CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# CHAOSS Website - Content, Issues, Requests and Edits
+
+The CHAOSS website is a Wordpress site.
+
+## Please use the Issues Tab to submit content requests, edits, and enhancements.
+Content requests could include upcoming events you would like to see posted on the site, project specific information, or requested edits.
+
+## News and blog posts
+To add news and blogs articles, please create a Markdown file in the news folder. When accepted, the content will be added to the website.
+
+## Wordpress/Github Markdown Content
+
+The site is built using GitHub content areas. Editing content in the associated GitHub markdown files will edit content on the Wordpress production site.
+
+Editable content are in folders in this repo and are named for the websites key navigation pages. Within these folders are pages and content areas in markdown files. The content areas are named using the header for the content section.
+
+To edit these content areas, please fork the repo and make changes to the markdown files.
```

Comment:
... and create a pull request. 

-- Without the pull request, we will not know about the changes.

Reply:
Maybe link to a .github/CONTRIBUTING.md file like other repos.

Reply:
This was implied.  I changed it to explicit.